<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddContentsClass</class>
 <widget class="QWidget" name="AddContentsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddContentsClass</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>특식</string>
    </attribute>
    <widget class="QListWidget" name="listWidget_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>국</string>
    </attribute>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>반찬</string>
    </attribute>
    <widget class="QListWidget" name="listWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>음식 이름</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>음식 종류</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>특식</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>국</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>반찬</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>종료</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>AddContentsClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
